modules/squants/src/main/scala/pureconfig/module/squants.scala

```diff
@@ -0,0 +1,179 @@
+  implicit val luminanceConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Luminance](Luminance.apply, _.toString)
+  implicit val luminousEnergyConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[LuminousEnergy](LuminousEnergy.apply, _.toString)
+  implicit val luminousExposureConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[LuminousExposure](LuminousExposure.apply, _.toString)
+  implicit val luminousFluxConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[LuminousFlux](LuminousFlux.apply, _.toString)
+  implicit val luminousIntensityConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[LuminousIntensity](LuminousIntensity.apply, _.toString)
+
+  // radio
+  implicit val irradianceConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Irradiance](Irradiance.apply, _.toString)
+  implicit val radianceConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Radiance](Radiance.apply, _.toString)
+  implicit val spectralIntensityConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SpectralIntensity](SpectralIntensity.apply, _.toString)
+  implicit val spectralIrradianceConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SpectralIrradiance](SpectralIrradiance.apply, _.toString)
+  implicit val spectralPowerConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SpectralPower](SpectralPower.apply, _.toString)
+
+  // space
+  implicit val angleConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Angle](Angle.apply, _.toString)
+  implicit val areaConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Area](Area.apply, _.toString)
+  implicit val lengthConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Length](Length.apply, _.toString)
+  implicit val solidAngleConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SolidAngle](SolidAngle.apply, _.toString)
+  implicit val volumeConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Volume](Volume.apply, _.toString)
+
+  // thermal
+  implicit val temperatureConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Temperature](parseTemperature, _.toString)
+  implicit val thermalCapacityConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[ThermalCapacity](ThermalCapacity.apply, _.toString)
+
+  // time
+  implicit val frequencyConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Frequency](Frequency.apply, _.toString)
+  implicit val timeConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Time](Time.apply, _.toString)
+
+  // This is temporary until https://github.com/typelevel/squants/pull/183 is released (1.2.0 ???)
+  // Without it, ScalaCheck has a pretty easy time breaking the conversion
+  private[squants] def parseTemperature(s: String): Try[Temperature] = {
+    val regex = "([-+]?[0-9]*\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)[ °]*(f|F|c|C|k|K|r|R)".r
+    s match {
+      case regex(value, unit) => unit match {
+        case Fahrenheit.symbol | "f" | "F" => Success(Fahrenheit(value.toDouble))
```

Comment:
Right. This is the current version in squants - https://github.com/typelevel/squants/blob/6affa29e75c3d5109f8345e2263af9a24ee35fff/shared/src/main/scala/squants/thermal/Temperature.scala#L143 :

```scala
def apply(s: String): Try[Temperature] = {
    val regex = "([-+]?[0-9]*\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)* *°? *(f|F|c|C|k|K|r|R)".r
    s match {
      case regex(value, unit) => unit match {
        case "f" | "F" => Success(Fahrenheit(value.toDouble))
        case "c" | "C" => Success(Celsius(value.toDouble))
        case "k" | "K" => Success(Kelvin(value.toDouble))
        case "r" | "R" => Success(Rankine(value.toDouble))
      }
      case _ => Failure(QuantityParseException("Unable to parse Temperature", s))
    }
  }
```

This version is probably going to be merged later today.